src/utils/locales.js

```diff
@@ -1,15 +1,18 @@
 import { addLocaleData } from 'react-intl';
 import en from 'react-intl/locale-data/en';
 import fr from 'react-intl/locale-data/fr';
+import zh from 'react-intl/locale-data/zh';
 
 import enTranslations from '../locales/en.json';
 import frTranslations from '../locales/fr.json';
+import zhTranslations from '../locales/zh.json';
 
-addLocaleData([...en, ...fr]);
+addLocaleData([...en, ...fr, ...zh]);
 
 export const translations = {
```

Comment:
Why are we duplicating the source of truth here?  Why not parse the locales directory for files matching `[a-z]{2}\.json` and building this dynamically? Otherwise everything needs to be done twice, once in that dir and again loaded here.

Reply:
This code is on the client side so it doesn't have access to the directory. However the available locales are defined in the file https://github.com/steemit/faucet/blob/master/helpers/locales.json
So I used this to import dynamically the locales.
Do this work for you ?

Reply:
yeah, looks great. a lot simpler. :)